### ISSUE TITLE
Angular must call super constructor in derived

### ISSUE THREAD

**Issue by adrian-wajs-pek** [swagger-api/swagger-ui#9326]
### Q&A (please complete the following information)
 - OS: Windows
 - Browser: All
 - Version:  5.9.0
 - Method of installation: npm
 - Swagger-UI version: 5.9.0
 - Swagger/OpenAPI version: 2.0

### Content & configuration

I am using angular and swagger-ui-dist library to add swagger-ui to my page.
While selecting enpoint I get error:
Could not render Vx, see the console.

And in console:
main.cc71d1475dd2fcfc1d8a.js:1 ReferenceError: Must call super constructor in derived class before accessing 'this' or returning from derived constructor

Swagger/OpenAPI definition:
NA

Swagger-UI configuration options:
{
      spec: api2,
      domNode: this.api2Element?.nativeElement,
      presets: [SwaggerUIBundle.presets.apis, SwaggerUIStandalonePreset.slice(1)],
      layout: 'StandaloneLayout',
      docExpansion: 'none',
      operationsSorter: 'alpha',
      tryItOutEnabled: false,
      deepLinking: true,
      displayOperationId: true,
      supportedSubmitMethods: [],
    }

### Screenshots

![image](https://github.com/swagger-api/swagger-ui/assets/148551358/ab7596d0-5e04-45d1-8392-cc095b911ad4)
![image](https://github.com/swagger-api/swagger-ui/assets/148551358/f0d643ae-4669-4e7d-baff-ec34ad310f8f)

### How can we help?

Provide solution.

**Comment by volkflo** [comment]
We are facing the same issue with angular@13 and the swagger-editor@4.11.X.
It worked before with swagger-editor@4.6.X.

Did you found a solution yet?

**Comment by adrian-wajs-pek** [comment]
Yes in my case it was version mismatch. I was using swaggerui with version: 4.18.3 and swagger-ui-dist: 5.9.0 and that was the reason it was not working for me.

**Comment by anandjaisy** [comment]
@adrian-wajs-pek  - I am facing the same issue with Angular 13 - I have just used `"swagger-ui-dist": "^5.13.0",`. What do you mean by **I was using swaggerui with version: 4.18.3 and swagger-ui-dist: 5.9.0** are you using two packages ??
